README.md

```diff
@@ -26,14 +85,72 @@ cd ape-chainstack
 python3 setup.py install
 ```
 
+Verify that the Chainstack plugin was installed correctly with this command:
+
+```bash
+ape plugins list
+```
+
+It will show a list of all the plugins installed, and Chainstack will be there.
+
+```bash
+Installed Plugins:
+  chainstack    <current version number>
+```
+
 ## Quick Usage
 
-Use in most commands using the `--network` option:
+Follow these steps to sign up on Chainstack, deploy a node, and find your endpoint credentials:
+
+1. [Sign up with Chainstack](https://console.chainstack.com/user/account/create).
+1. [Deploy a node](https://docs.chainstack.com/platform/join-a-public-network).
+1. [View node access and credentials](https://docs.chainstack.com/platform/view-node-access-and-credentials).
+
+> **Note:** At this moment only the Ethereum network is supported.
+
+Create an environment variable with your Chainstack node URL in this format `CHAINSTACK_"NETWORK"_URL=ENDPOINT_URL`; for example:
+
+```sh
+export CHAINSTACK_GOERLI_URL=https://nd-11X-26X-16X.p2pify.com/YOUR_API_KEY
+```
+
+Use the command `ape networks list` to see the networks available:
+
+```sh
+ethereum  (default)
+├── mainnet
+│   ├── geth  (default)
+│   └── chainstack
+├── ropsten
+│   ├── geth  (default)
+│   └── chainstack
+├── kovan
+│   └── geth  (default)
+├── rinkeby
+│   ├── geth  (default)
+│   └── chainstack
+├── goerli
+│   ├── geth  (default)
+│   └── chainstack
+└── local  (default)
+    ├── geth
+    └── test  (default)
+```
+
+Use the `--network` command to access the console using your node; for example:
 
 ```bash
-ape console --network ethereum:ropsten:chainstack
+ape console --network ethereum:goerli:chainstack
 ```
 
+This is a list and syntax of the networks available once you install the Chainstack plugin.
+
+```bash
```

Comment:
Consider an explanation instead of enumerating all the options here. Something like:

```
You compose the network option by following this formula: <ecosystem>:<network>:<provider> where "chainstack" is the provider.
See the full output of networks in your ape instance by using the command `ape networks list`.
For more information on networking un ape, see [this guide (https://docs.apeworx.io/ape/stable/userguides/networks.html).
```

Let's say we increase support for new networks in chainstack; this list will become outdated. I am trying to avoid someone having to keep this documentation up to date as the plugin grows.

Reply:
Linking to this doc is probably sufficient https://docs.apeworx.io/ape/stable/userguides/networks.html

Reply:
true, you can go ahead and propose the change if you can, it's probably quicker like that than if I change it later

Reply:
I actually updated it in on my side. What is a good way to implement it here now?